2020-02-Spring/0_arrays_matrix/README.md

```diff
@@ -141,21 +153,122 @@ Go to [Top](#top)
 <!-- Don't remove -->
 <a name="s3"/>
 
-### 3. SOLUTION 3 TODO :bug:
-
-Source: TODO :bug:
-
-#### Naive/Simple Solution 
-
-TODO :bug:
-
-#### Optimal Solution
-
-TODO :bug:
+### 3. Word Search Solution 
+
+#### Solution
+
+
+Time Complexity: O(N * 4^{L}) where N is the number of cells in the board and L is the length of the word to be matched.
```

Comment:
Monospacing is usually nice for complexity results IMO, but feel free to overrule me
```suggestion
Time Complexity: `O(N * 4^{L})` where `N` is the number of cells in the board and `L` is the length of the word to be matched.
```